Queueomatic/Server/Endpoints/ResetPassword/ResetPasswordEndpoint.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FastEndpoints;
+using Queueomatic.DataAccess.UnitOfWork;
+using Queueomatic.Server.Services.AuthenticationService;
+
+namespace Queueomatic.Server.Endpoints.ResetPassword;
+
+public class ResetPasswordEndpoint : Endpoint<ResetPasswordRequest>
+{
+    private readonly IUnitOfWork _unitOfWork;
+    private readonly IAuthenticationService _authenticationService;
+
+    public ResetPasswordEndpoint(IUnitOfWork unitOfWork, IAuthenticationService authenticationService)
+    {
+        _unitOfWork = unitOfWork;
+        _authenticationService = authenticationService;
+    }
+
+    public override void Configure()
+    {
+        Post("/resetPassword");
```

Comment:
The path should be changed to `/resetPassword/{token}`

Reply:
Why is this?

Reply:
Because this path does not match the link we send out.